drivers/timer/mcux_os_timer.c

```diff
@@ -88,6 +88,19 @@ void mcux_lpc_ostick_isr(const void *arg)
 }
 
 #if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(standby)) && CONFIG_PM
+#define MCUX_OS_TIMER_COUNTER_CONFIG_INIT(...)	\
+{						\
+	.callback = NULL,			\
+	.user_data = NULL,			\
+	.flags = 0,				\
+}
+
+/* the same cfg can't be used because the different types
+ * of counter config structs have the fields in different orders, of course.
+ */
+struct counter_top_cfg top_cfg = MCUX_OS_TIMER_COUNTER_CONFIG_INIT();
+struct counter_alarm_cfg alarm_cfg = MCUX_OS_TIMER_COUNTER_CONFIG_INIT();
```

Comment:
Why not change this to 
struct counter_top_cfg top_cfg = { 0 }
struct counter_alarm_cfg alarm_cfg = { 0 }

Reply:
makes sense